snapshots.yml

```diff
@@ -15,3 +15,4 @@
   url: /news
 - name: Article
   url: /news/2019/09/demo-article
+  waitForTimeout: 1000
```

Comment:
Oh, it doesn't look like this got the mobile menu to show up in Percy. Too bad. That can be investigated in another issue I guess.